src/vm/core.zig

```diff
@@ -1181,93 +1181,13 @@ pub fn computeRes(
 ) !?MaybeRelocatable {
     return switch (instruction.res_logic) {
         .Op1 => op_1,
-        .Add => try addOperands(op_0, op_1),
-        .Mul => try mulOperands(op_0, op_1),
+        // .Add => try addOperands(op_0, op_1),
```

Comment:
i believe this is removable :)

Reply:
Perfectly right, I've just removed it :)